src/integrationtests/java/com/aws/greengrass/integrationtests/deployment/DeploymentServiceIntegrationTest.java

```diff
@@ -247,9 +260,9 @@ void WHEN_multiple_local_deployment_scheduled_THEN_all_deployments_succeed() thr
 
         submitLocalDocument(request);
 
-        firstDeploymentCDL.await(10, TimeUnit.SECONDS);
-        secondDeploymentCDL.await(10, TimeUnit.SECONDS);
-        thirdDeploymentCDL.await(10, TimeUnit.SECONDS);
+        assertTrue(firstDeploymentCDL.await(10, TimeUnit.SECONDS));
```

Comment:
nit

add a message so we more easily know which one failed instead of only going by the line number.

Reply:
added